go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/buildpacks/libcnb
 go 1.17
 
 require (
+	github.com/CycloneDX/cyclonedx-go v0.6.0
```

Comment:
I'm reluctant to have this package added. Especially because it is just for example code. Can we do without this? Or can someone confirm that Go is smart enough to prune this out of the resulting buildpack binary image?

Reply:
Go is smart enough to prune unused dependencies.  Writing a trivial `main.go` I can then run `go version` to see the dependencies.

```
$ go version -m main
main: go1.18.1
	path	command-line-arguments
	dep	github.com/BurntSushi/toml	v1.2.0	h1:Rt8g24XnyGTyglgET/PRUNlrUeu9F5L+7FilkXfZgs0=
	dep	github.com/Masterminds/semver	v1.5.0	h1:H65muMkzWKEuNDnfl9d70GUjFniHKHRbFPGBuZ3QEww=
	dep	github.com/buildpacks/libcnb	(devel)
	dep	github.com/onsi/gomega	v1.20.0	h1:8W0cWlwFkflGPLltQvLRB7ZVD5HuP6ng320w2IS245Q=
	build	-compiler=gc
	build	CGO_ENABLED=1
	build	CGO_CFLAGS=
	build	CGO_CPPFLAGS=
	build	CGO_CXXFLAGS=
	build	CGO_LDFLAGS=
	build	GOARCH=arm64
	build	GOOS=darwin
```